### ISSUE TITLE
Applied ./start.sh again accidentally, how to recover data?

### ISSUE THREAD

**Issue by naveen-appoets** [synechron-finlabs/quorum-maker#84]
So I wanted to restart the node and hence switched to the directory that was created for the node and gave start.sh and the data is now all gone. Anyway to recover it?

**Comment by dhyansraj** [comment]
./start.sh shouldnt delete the data. It is there to start/restart the node. Probably the node not connecting to other nodes. Can you check the other nodes and see if this is part of the network. Also see geth log possible errors.

**Comment by dhyansraj** [comment]
It should still show the old blocks even if there are no peers available currently. Let me try to reproduce and get back to you. I am assume you were able to see both nodes in sync when they were online.

**Comment by naveen-appoets** [comment]
Yes, I was able to see both nodes in sync when they were both online. Stopping one node unconventionally (let's say server got shutdown) made the other node not produce any more blocks. Restarting the node showed block count 0.
But yes, I did not remove the lost peer before restarting.

**Comment by dhyansraj** [comment]
you invoked ./start.sh from inside the nodename directory, right ?

**Comment by naveen-appoets** [comment]
Yeah

**Comment by melnikaite** [comment]
I have the same issue with local installation
I just generated network using `./setup.sh dev -p TestNetwork -n 3 -e`
Run `docker-compose up -d` inside `TestNetwork`
Deployed some contracts
Stopped containers `docker-compose stop`
Then started again `docker-compose up -d`
All nodes shows that chain has just 1 block in explorer and rpc with all previous transactions
Also all ABIs disappeared
Image `syneblock/quorum-maker:2.2.0_2.6.1`

**Comment by dhyansraj** [comment]
Can you try with docker-compose down instead of stop. ?

**Comment by melnikaite** [comment]
`docker-compose down` take the same effect

**Comment by muhammetevirgen** [comment]
So if we try to test some disaster issue and reboot the machine , how it will behave ?

it will behave like docker-compose down ? or docker-compose stop ? or none of them ?

Because when I run start.sh on distributed network after reboot ,my node is not syncing

**Comment by dhyansraj** [comment]
Is this problem happening consistently ? Which version of QM are you using? I tried in couple of different VMs including RehHat and Ubuntu. Not able to reproduce.

**Comment by melnikaite** [comment]
QM latest master 21c0d1a41af9d5bd686e460ceb1143f02858dca5
OS X 10.14.2
docker 18.09.1
docker-compose 1.23.2
docker image syneblock/quorum-maker:2.2.0_2.6.1

The issue is constantly reproducible

**Comment by dhyansraj** [comment]
I think your issue is similar to #87 and is fixed now. Can you try again?
